tests/periph_i2c/tests/test.py

```diff
@@ -0,0 +1,328 @@
+import periph_i2c_if
+import bpt_if
+import logging
+import argparse
+import time
+
+__BPT_ADDR = 85
+__BPT_USER_REG = 152
+
+
+def kwexpect(val1, val2, level="WARN"):
+    res = level
+    try:
+        if (isinstance(val1, str) and isinstance(val2, str)):
+            if (val1.lower() == val2.lower()):
+                res = "PASS"
+        elif (isinstance(val1, list)):
+            err = False
+            if (len(val1) != len(val2)):
+                err = True
+            for x in range(0, len(val1)):
+                if (val1[x] != val2[x]):
+                    err = True
+            if (err is False):
+                res = "PASS"
+            else:
+                return [level, val1, val2]
+        elif ((val1 == val2) or (val1 is val2)):
+            res = "PASS"
+    except TypeError:
+        res = 'TypeError'
+
+    return [res, val1, val2]
+
+
+class TestParam:
```

Comment:
might also wanna use dictionary here?

Reply:
Yup but I don't want to spend more time on this.